### DISCUSSION TITLE
Readme file images randomly no longer display

### DISCUSSION THREAD

**Discussion by micbay** [community/community#70859] [Repositories] [answered]
### Select Topic Area

Bug

### Body

I have a readme file in my repo that has not been changed in a while, but suddenly some of the images are no longer visible, while other images still display fine.

When inspecting the affected repository's home page, the missing images appear as 404 not found, even though they still exist, and are located inside a sub-folder of the repo called 'images'. It appears that, for some reason, github is parsing the markdown differently for different images.

In the raw markdown, all image references are using the same relative path reference of `Images/image_of_interest`. It appears that for the broken images, github is building the image path using my github user name url as the root path ( `github/user/Images`), but for the working images it is properly, still using the repo url as the root for the relative path, ie `github/user/repo/Images`. It's not clear to me from the raw markdown why they are being processed differently.

The issue appears to be caused by something github does in presenting repository readme files. Whatever is causing the issue, it seems to be due to a recent change to the page display scripts used, as no changes to the markdown has occurred between all images displaying vs now.

Here are the errors of the missing images:
![github readme images not viewable](https://github.com/community/community/assets/23461494/b537da8f-0ba7-485f-aaee-0ded1ab45dea)

For specific reference of a couple of the missing images, please see the broken links in the ['Noise Sensitivity' section ](https://github.com/micbay/RaceTimerAndController#noise-sensitivity)of the [problem repo](https://github.com/micbay/RaceTimerAndController).

**Comment by seyLu** [comment]
#### Hi @micbay,

It is a currently known bug that relative links don't work on blockquotes ([see related discussion](https://github.com/orgs/community/discussions/67750)). 

As a workaround, you can either remove the image from the blockquote or use absolute link, eg:
```bash
https://github.com/micbay/RaceTimerAndController/blob/master/Images/C1.png
```

**Comment by VanshGehlot** [comment]
It sounds like you are running into an issue where GitHub is inconsistently resolving relative image links in your README file. A few things to try:

1. Double check that all your image links use a consistent relative path like Images/image.png. Make sure there are no typos or inconsistencies.
2. Try using absolute links to the images instead of relative links. This will avoid any ambiguity with how GitHub resolves the paths.
3. If using relative links, make sure the image folder is in the same directory as the README file. Don't link across folders with ../ relative paths.
4. Check if you have any special characters or spacing in the image filenames that could be getting parsed incorrectly. Try simplifying the names.
5. Clear your browser cache and reload the page. Sometimes cached files can cause display issues.
6. Try viewing the README rendered on the main page vs raw. The rendering logic may differ.
7. As a last resort, you can migrate your images to a hosted image service like Imgur and link to them instead of self-hosting.

Hopefully one of those suggestions helps resolve the inconsistent image display problem you're running into.

**Comment by mdtanvirahamedshanto** [comment]
Hi

**Comment by Narigo** [comment]
**Edit:** Sorry, the "search for similar discussions" brought me here and I thought this was about mobile as well. I seem to have a similar issue, but mine was [specifically about images not showing up in the GitHub mobile app when using relative images](https://github.com/orgs/community/discussions/72093).

**Comment by micbay** [comment, marked as answer]
Just to conclude this topic, the github script rendering markdown, appears to have been updated to again properly, display relative image links, regardless of the markdown element type and this bug is no longer an issue.
